deployment.go

```diff
@@ -99,6 +100,12 @@ func (d *Deployment) restartServices(services []string, failuresPermitted, timeo
 		return nil
 	}
 
+	// Don't restart services when a lock is present
+	if _, err := os.Stat("/etc/service/dont-restart"); err == nil {
```

Comment:
Sorry, I just noticed this now - could you put the lock file in a different location? `/etc/service/don-restart` is totally non-obvious. What about `/etc/sv-rollout/no-restart`?